packages/host/src/AppDrawer.test.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import AppDrawer from './AppDrawer';
+import { AppThemeProvider } from '@openmsupply-client/common';
+import { BrowserRouter } from 'react-router-dom';
+import { IntlTestProvider } from '@openmsupply-client/common/src/intl';
+import { act } from 'react-dom/test-utils';
+
+describe('AppDrawer', () => {
+  it('Expands when clicking the expand button', () => {
+    render(
+      <AppThemeProvider>
+        <IntlTestProvider>
+          <BrowserRouter>
+            <AppDrawer />
+          </BrowserRouter>
+        </IntlTestProvider>
+      </AppThemeProvider>
+    );
+
+    const button = screen.getByRole('button', { name: /Open the menu/i });
```

Comment:
it is certainly difficult to create tests which aren't fragile, that's for sure. If we change this label the test breaks which isn't great!